### PR TITLE
Kintsugi & Kiln Support

### PR DESCRIPTION
**Pull request by CarlBeek**
Adds support for the Kintsugi and Kiln merge testnets & deprecates support for Pyrmont.

### CODE REVIEW

File: staking_deposit/settings.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 from typing import Dict, NamedTuple
 
 
-DEPOSIT_CLI_VERSION = '1.2.0'
+DEPOSIT_CLI_VERSION = '2.0.0'
```
Comment (hwwhww) [inline staking_deposit/settings.py:4]: [not strongly] nitpicking it's better to be in another PR.

**Review by hwwhww — APPROVED** [review]

**Comment by hwwhww** [comment]
replace #217

File: staking_deposit/settings.py (hunk 1 above)
Comment (CarlBeek) [reply staking_deposit/settings.py]: 👍 agreed it was a lazy move. I have reverted it in https://github.com/ethereum/eth2.0-deposit-cli/pull/240/commits/e562b5f8cd19513460d39d18fc86213a6164036d

will open new PR to adress

**Review by rsalvo77 — COMMENTED** [review]
:)  TY
